DESCRIPTION

```diff
@@ -58,6 +57,7 @@ Imports:
 Suggests:
     austraits,
     bench,
+    data.table,
```

Comment:
Can we please remove datat.table. Find whatever function uses it and an alternative from existing packages. It's quite a heavy decency and I'd rather not include it.